test/e2e-ansible/e2e_ansible_olm_test.go

```diff
@@ -40,12 +37,6 @@ var _ = Describe("Integrating ansible Projects with OLM", func() {
 			OLMStatusDescriptorsTest  = "olm-status-descriptors"
 		)
 
-		BeforeEach(func() {
-			By("turning off interactive prompts for all generation tasks.")
-			replace := "operator-sdk generate kustomize manifests"
-			testutils.ReplaceInFile(filepath.Join(tc.Dir, "Makefile"), replace, replace+" --interactive=false")
-		})
```

Comment:
this code was replaced by in the suite test only. 

```go 
By("turning off interactive prompts for all generation tasks.")
err = tc.DisableOLMBundleInteractiveMode()
Expect(err).NotTo(HaveOccurred())
```

Reply:
:+1: